icu4c/source/test/testdata/rbbitst.txt

```diff
@@ -1898,52 +1898,19 @@ Bangkok)•</data>
 
 <locale ja@lw=phrase>
 <line>
```

Comment:
We need a way to distinguish between test data for the existing line breaking implementation (so that we don't lose existing coverage) vs. the new implementation. Maybe `<line>` for the existing implementation, and a separate set of data after, say, `<lineML>` for the new implementation. The test code would need to use vs. ignore one or the other.

Reply:
Done. Thanks for the suggestion.

Reply:
Some questions - can the old phrase line break and new ML based implementations coexist in an ICU build? Is it a build or a run-time option to specify which one you get? How is the choice specified?

(I'm sorry, I haven't really looked into this yet)

If possible, I'd prefer to not add a new top level tag to the test data syntax. Or, if it is necessary, to have something more general that could handle similar conditional situations that may happen later on.

Reply:
They can coexist.
I add a compile flag, UCONFIG_NO_CJ_ML_PHRASE_BREAKING, in uconfig.h.